tools/manifest-templator/manifest-templator.go

```diff
@@ -188,7 +189,17 @@ func getOperatorDeploymentSpec(data templateData) string {
 	if err != nil {
 		panic(err)
 	}
-	return fixResourceString(writer.String(), 12)
+	spec := writer.String()
+
+	// remove creationTimestamp
```

Comment:
could be nice to add a line explaining why - it's just a operatorhub.io current limitation? (I've read https://github.com/operator-framework/community-operators/pull/314 but still non-obvious)

Reply:
I understood that having a null creationTimestamp just does not look nice (and I completely agree with it :wink: )